### PR TITLE
Bump several dev dependencies, Ember CLI 3.26.1

### PR DESCRIPTION
**Pull request by mixonic**
There is very little in terms of end-user impact from this set of changes. I called out a few things below.

I'll suggest that after this lands we should cut 1.2.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,9 +12,9 @@ integration when that feature is enabled.
 Compatibility
 ------------------------------------------------------------------------------
 
-* Ember.js v2.18 or above
+* Ember.js v2.16 or above
 * Ember CLI v2.13 or above
-* Node.js v8 or above
+* Node.js v10 or above
```
Comment (mixonic) [inline README.md:17]: This simply updated the README to reflect reality of `master`.

File: package.json [hunk 2]
```diff
@@ -27,43 +30,50 @@
   "dependencies": {
     "broccoli-funnel": "^2.0.2",
     "broccoli-merge-trees": "^3.0.2",
-    "ember-cli-babel": "^7.18.0",
+    "ember-cli-babel": "^7.26.3",
```
Comment (mixonic) [inline package.json:33]: Actual dependency bump.

**Review by simonihmig — APPROVED** [review]

File: config/ember-try.js [hunk 3]
```diff
@@ -130,7 +135,9 @@ module.exports = function() {
             },
           },
         },
-      ]
+        embroiderSafe(),
+        embroiderOptimized(),
```
Comment (simonihmig) [inline config/ember-try.js:139]: These scenarios need to be added to the GH action matrix!

**Review by simonihmig — COMMENTED** [review]
Just realized there is one little piece missing...
(which I also forgot in my preceding PR 🙈 )

File: config/ember-try.js (hunk 3 above)
Comment (mixonic) [reply config/ember-try.js:139]: Yes! I swear I actually ran `ember-safe` locally too, but then forgot to add it.

I have pushed an update which adds `embroider-safe` to the suite which passes. `embroider-optimized` does not pass, and I've simply left it out of the GH workflow for now.

**Pull request merged**